db/migrate/20230213141824_add_true_layer_provider_id_applicant_index_to_bank_providers.rb

```diff
@@ -0,0 +1,7 @@
+class AddTrueLayerProviderIdApplicantIndexToBankProviders < ActiveRecord::Migration[7.0]
+  disable_ddl_transaction!
```

Comment:
Our applicant table is small, so it probably wouldn't be a problem, but adding indexes blocks writes —so we want to `disable_ddl_transaction` and use the `concurrently` algorithm.

https://thoughtbot.com/blog/how-to-create-postgres-indexes-concurrently-in

Reply:
👍